client/src/pages/Login/Login.tsx

```diff
@@ -16,6 +16,18 @@ export default function Login(): JSX.Element {
   const { updateLoginContext } = useAuth();
   const { updateSnackBarMessage } = useSnackBar();
 
+  const handleDemoLogin = () => {
+    demoLogin('demo@kanban.com', '123456').then((data) => {
+      if (data.error) {
+        updateSnackBarMessage(data.error.message);
+      } else if (data.success) {
```

Comment:
Would have liked to not expose the test account credentials in front-end, as I said in the sprint we should have a route on the back-end that will deal with logging in the demo user and uses environment vars for the email/password. This would have been better.